### DISCUSSION TITLE
Unique with where condition

### DISCUSSION THREAD

**Discussion by RogerBrusamarello** [prisma/prisma#24485] [Q&A]
### Question

There is a way to create a unique index with where condition.

Like this, but directly in schema part:

```
-- CreateTable
CREATE TABLE "organizations" (
    "id" TEXT NOT NULL,
    "name" TEXT NOT NULL,
    "cnpj" TEXT NOT NULL,
    "abbreviation" VARCHAR(3) NOT NULL,
    "email" TEXT,
    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
    "updatedAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
    "deletedAt" TIMESTAMP(3),

    CONSTRAINT "organizations_pkey" PRIMARY KEY ("id")
);


-- CreateIndex
CREATE UNIQUE INDEX "organizations_name_key" ON "organizations"("name") WHERE "deletedAt" IS NULL;
```
### Expected behavior (optional)

Add some kind tag on schema to add automatically where condition on CREATE UNIQUE INDEX

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
model Organization {
  id           String      @id @default(uuid())
  name         String      @unique
  cnpj         String      @unique
  abbreviation String      @unique @db.VarChar(3)
  email        String?
  createdAt    DateTime    @default(now())
  updatedAt    DateTime    @default(now()) @updatedAt
  deletedAt    DateTime?
  
  @@map("organizations")
  }
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by janpio** [comment]
Feature request to support this in Prisma Schema language is tracked here: https://github.com/prisma/prisma/issues/6974

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
